src/rowbinary/ser.rs

```diff
@@ -114,9 +309,17 @@ impl<'a, B: BufMut> Serializer for &'a mut RowBinarySerializer<B> {
     #[inline]
     fn serialize_newtype_struct<T: Serialize + ?Sized>(
         self,
-        _name: &'static str,
+        name: &'static str,
         value: &T,
     ) -> Result<()> {
+        // This type is special cased to force raw byte serialization without length
+        // prefix. See RawBinary for more info.
+        if name == RAW_BINARY_NAME_TAG {
```

Comment:
Is this some hacky way to specialize how this is serialized or does this code generally rely on these magic tag names?

Reply:
It's a hacky way to do it. I can't figure out any better way. If you know if something let me know.

Reply:
https://stackoverflow.com/questions/69045460/serde-custom-serializer